addons/web/static/src/views/form/form_controller.js

```diff
@@ -94,7 +96,14 @@ export class FormController extends Component {
         this.user = useService("user");
         this.viewService = useService("view");
         this.ui = useService("ui");
+        this.state = useState({
+            isDisabled: false,
+        });
         useBus(this.ui.bus, "resize", this.render);
+        useFormErrorDialog(async () => {
+            await this.discard();
+            this.env.config.historyBack();
```

Comment:
historyBack? I may not understand everything about these lines, but we only want to do an history back in some cases (e.g. list view -> create -> save (fails) -> go back to list). We must be sure that we do not do an history back each time a save fails